apps/accounts/admin.py

```diff
@@ -161,21 +175,39 @@ def send_button(self, obj):
     send_button.short_description = 'Send email'
 
     def send_email(self, request, *args, **kwargs):
-        email_template = request.GET.get('email')
-        email_template = f'emails/{email_template}'
+        email_name = request.GET.get('email')
+        email_template = f'emails/{email_name}'
+        redirect_name = 'admin:' + get_admin_name(self.model, 'change')
```

Comment:
I've seen three different ways of formatting strings in the last three pull requests, and it's not stopping functionality, but it is a bit inconsistent. I've created an issue to agree on style for this reason, to discuss in future,

https://github.com/thegreenwebfoundation/greenwebfoundation-admin/issues/23

Reply:
Good point.